File: Code Generation/responses/cpp/claude/easy/maximum-students-on-a-single-bench.cpp
#include <vector>
#include <unordered_map>
#include <unordered_set>
#include <algorithm>
using namespace std;

class Solution {
public:
    int maxStudentsOnBench(vector<vector<int>> students) {
        unordered_map<int, unordered_set<int>> bench;
        for (auto& s : students)
            bench[s[1]].insert(s[0]);
        int res = 0;
        for (auto& [b, st] : bench)
            res = max(res, (int)st.size());
        return res;
    }
};

File: Code Generation/responses/python/claude/easy/maximum-students-on-a-single-bench.py
from typing import List
from collections import defaultdict

class Solution:
    def maxStudentsOnBench(self, students: List[List[int]]) -> int:
        bench = defaultdict(set)
        for student_id, bench_id in students:
            bench[bench_id].add(student_id)
        return max((len(s) for s in bench.values()), default=0)